src/main/java/seedu/address/logic/commands/FindConcertContactCommand.java

```diff
@@ -0,0 +1,136 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CONCERT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PERSON;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_CONCERT_CONTACTS;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.concert.Concert;
+import seedu.address.model.concert.ConcertContact;
+import seedu.address.model.person.Person;
+
+/**
+ * Finds and lists all concertContacts in address book
+ * which is associated to the person keyword
+ * and is associated to the concert keyword.
+ * Keyword matching is case-insensitive.
+ */
+public class FindConcertContactCommand extends Command {
+
+    public static final String COMMAND_WORD = "findcc";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all concertContacts which are associated to "
+            + "the person at the specified person index and the concert at the specified concert index "
+            + "and displays them as a list with index numbers.\n"
+            + "Parameters: "
+            + "[" + PREFIX_PERSON + "PERSON_INDEX] "
+            + "[" + PREFIX_CONCERT + "CONCERT_INDEX]\n"
```

Comment:
Is there a way to show that at least one of PERSON_INDEX or CONCERT_INDEX have to be specified

Reply:
I couldn't think of a way to do it here, but we should include this condition in the user guide

Reply:
I included this line to the message usage "The person or concert index, or both of them must be included."